phoenix-core/src/test/java/org/apache/phoenix/cache/ServerMetadataCacheIT.java

```diff
@@ -1700,9 +1704,109 @@ public void testInheritedIndexOnTenantViews(boolean sameTenantViewNames) throws
         }
     }
 
+    /**
+     * Test that a client can learn about the changes to a base table's
+     * sequence number after a view is created on the table.
+     */
+    @Test
+    public void testBaseTableSequenceNumberAfterCreateView() throws Exception {
```

Comment:
Do we need this test? AlterTableIT#testAddingColumnsToTablesAndViews does the exact same thing.